util/grpc/src/lib.rs

```diff
@@ -75,23 +75,38 @@ pub fn send_result<T>(
     SVC_COUNTERS.resp(&ctx, success);
 }
 
+macro_rules! report_err_with_code(
```

Comment:
This was changed to a macro to deal with `slog`'s macro hell (the underlying logging crate we use).